app/Module/GoogleMapsModule.php

```diff
@@ -233,7 +230,7 @@ public function getBoxChartMenu(Individual $individual) {
 	 * A form to edit the module configuration.
 	 */
 	private function config() {
-		$controller = new PageController;
+		$controller = new PageController();
```

Comment:
Did PhpStorm do this for you, or are you proposing to change the code style.

I note that we are currently inconsistent in this.  I *think* I prefer `new foo;` to `new foo();`.

But if we are going to reformat all of these, it should probably happen in a separate PR.